kmod/src/inode.c

```diff
@@ -808,15 +831,18 @@ static void store_inode(struct scoutfs_inode *cinode, struct inode *inode)
 int scoutfs_dirty_inode_item(struct inode *inode, struct scoutfs_lock *lock)
 {
 	struct super_block *sb = inode->i_sb;
-	struct scoutfs_inode sinode;
+	struct scoutfs_sb_info *sbi = SCOUTFS_SB(sb);
+	struct scoutfs_inode_payload_wrapper wrap;
 	struct scoutfs_key key;
+	int inode_bytes;
 	int ret;
 
-	store_inode(&sinode, inode);
+	inode_bytes = scoutfs_inode_fmt_ver_bytes(sbi->fmt_vers);
```

Comment:
I wonder if this should take an sb to save all the callers from having a local sbi that they only use to pass in.   If we're only ever using it to determine the mounted bytes in use, and not the bytes for any arbitrary version, that seems reasonable.